checks/smtp/go.mod

```
@@ -0,0 +1,11 @@
+module github.com/alexliesenfeld/health/checks/smtp
+
+go 1.19
```

Can we reduce this to go 1.13? This library supports Go versions down to 1.13 as of now, so it would be good if we could keep 1.13.
```suggestion
go 1.13
```